frontend/packages/shared/src/contexts/ServicesContext.test.tsx

```diff
@@ -12,13 +29,45 @@ const wrapper = ({ children }) => (
 );
 
 describe('ServicesContext', () => {
+  it('displays a specific error message if API returns an error code and the error messages does exist', async () => {
+    const mockConsoleError = jest.spyOn(console, 'error').mockImplementation();
+    const { result } = renderHook(() => useQuery(['fetchData'],() => Promise.reject({
+      response: {
+        data: {
+          errorCode: 'DM_01',
+        }
+      }
```

Comment:
What do you think of creating a utility method, `createServerErrorResponse(errorCode: string): ErrorResponse`? This might make it easier to mock errors without remembering the object structure. 🤔 If the object structure changes in the future we do not need to change it in every tests, but just the utility method.

Reply:
Good idea 👍 Done!